introduction.md

```diff
@@ -0,0 +1,5 @@
+*My name is Daisuke Takeshita. I study the retina.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?